src/access/examples/ExampleAccessControlDelegate.cpp

```diff
@@ -483,6 +512,99 @@ class EntryStorage
         index = found ? toIndex : ArraySize(acl);
     }
 
+public:
+    static constexpr uint8_t kTagInUse       = 1;
+    static constexpr uint8_t kTagFabricIndex = 2;
+    static constexpr uint8_t kTagAuthMode    = 3;
+    static constexpr uint8_t kTagPrivilege   = 4;
+    static constexpr uint8_t kTagSubjects    = 5;
+    static constexpr uint8_t kTagTargets     = 6;
+    static const size_t kStorageBufferSize   = 192;
+
+    CHIP_ERROR Serialize(chip::PersistentStorageDelegate * storage, const char * key)
+    {
+        uint8_t buffer[kStorageBufferSize] = { 0 };
+        chip::TLV::TLVWriter writer;
+        writer.Init(buffer);
+        chip::TLV::TLVType container;
+        ReturnErrorOnFailure(writer.StartContainer(chip::TLV::AnonymousTag(), chip::TLV::TLVType::kTLVType_Structure, container));
+
+        ReturnErrorOnFailure(writer.Put(chip::TLV::ContextTag(kTagInUse), mInUse));
+        ReturnErrorOnFailure(writer.Put(chip::TLV::ContextTag(kTagFabricIndex), mFabricIndex));
+        ReturnErrorOnFailure(writer.Put(chip::TLV::ContextTag(kTagAuthMode), mAuthMode));
+        ReturnErrorOnFailure(writer.Put(chip::TLV::ContextTag(kTagPrivilege), mPrivilege));
+
+        chip::TLV::TLVType internalContainer;
+        ReturnErrorOnFailure(
+            writer.StartContainer(chip::TLV::ContextTag(kTagSubjects), chip::TLV::TLVType::kTLVType_Array, internalContainer));
+        for (size_t i = 0; i < kMaxSubjects; ++i)
+        {
+            ReturnErrorOnFailure(mSubjects[i].Serialize(writer));
+        }
+        ReturnErrorOnFailure(writer.EndContainer(internalContainer));
+
+        ReturnErrorOnFailure(
+            writer.StartContainer(chip::TLV::ContextTag(kTagTargets), chip::TLV::TLVType::kTLVType_Array, internalContainer));
+        for (size_t i = 0; i < kMaxTargets; ++i)
+        {
+            ReturnErrorOnFailure(mTargets[i].Serialize(writer));
+        }
+        ReturnErrorOnFailure(writer.EndContainer(internalContainer));
+
+        ReturnErrorOnFailure(writer.EndContainer(container));
+        ReturnErrorOnFailure(writer.Finalize());
+
+        return storage->SyncSetKeyValue(key, buffer, static_cast<uint16_t>(writer.GetLengthWritten()));
+    }
+
+    CHIP_ERROR Deserialize(chip::PersistentStorageDelegate * storage, const char * key)
+    {
+        uint8_t buffer[kStorageBufferSize] = { 0 };
+        uint16_t bufferSize                = static_cast<uint16_t>(sizeof(buffer));
+        ReturnErrorOnFailure(storage->SyncGetKeyValue(key, buffer, bufferSize));
+        chip::TLV::TLVReader reader;
+        reader.Init(buffer, bufferSize);
+
+        ReturnErrorOnFailure(reader.Next(chip::TLV::TLVType::kTLVType_Structure, chip::TLV::AnonymousTag()));
+
+        chip::TLV::TLVType container;
+        ReturnErrorOnFailure(reader.EnterContainer(container));
+
+        // mInUse
```

Comment:
Consider dropping these comments - they seem to just restate the code below them.

Reply:
Done.